crypto/sign_norelic.go

```diff
@@ -10,3 +11,18 @@ func newSigner(algo SigningAlgorithm) (signer, error) {
 func init() {
 	initNonRelic()
 }
+
+// VerifyPOP verifies a proof of possession (PoP) for the receiver public key; currently only works for BLS
+func VerifyPOP(pk PublicKey, s Signature) (bool, error) {
+	panic("VerifyPOP not supported with non-relic build")
+}
+
+// AggregateSignatures aggregate multiple signatures into one; currently only works for BLS
+func AggregateSignatures(sigs [][]byte) (Signature, error) {
+	panic("AggregateSignatures not supported with non-relic build")
+}
+
+// AggregatePublicKeys aggregate multiple public keys into one; currently only works for BLS
+func AggregatePublicKeys(keys []PublicKey) (PublicKey, error) {
+	panic("AggregatePublicKeys not supported with non-relic build")
+}
```

Comment:
In the crypto package, these function should not be exported when Relic tag is not enabled.
I understand the need in FVM is different (these functions should always be defined). Since it is an FMV issue, I suggest we define these functions (relic and non-relic) in `fvm/crypto`. This is already done for BLS KMAC ([relic](https://github.com/onflow/flow-go/blob/cadence-issue-1127/fvm/crypto/blskmac_relic.go) and [non-relic](https://github.com/onflow/flow-go/blob/cadence-issue-1127/fvm/crypto/blskmac.go)).
The files can be renamed to reflect that they now contain multiple crypto tools and not just BLS_KMAC.